Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN npm run build -ws
 # Production Stage #
 ####################
 
-FROM node:16.19.1-bullseye-slim
+FROM node:21.7.1-bullseye-slim
```

Comment:
I noticed fetch worked on my local but not when deploying. The old node version might have been the culprit :o 

https://stackoverflow.com/questions/48433783/referenceerror-fetch-is-not-defined